### PR TITLE
Write errors to the cluster status

### PR DESCRIPTION
**Pull request by dAdAbird**

### CODE REVIEW

**Comment by hors** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-182-98a3300

File: pkg/controller/pxc/controller.go [hunk 1]
```diff
@@ -111,16 +111,25 @@ func (r *ReconcilePerconaXtraDBCluster) Reconcile(request reconcile.Request) (re
 		return reconcile.Result{}, err
 	}
 
+	defer func() {
+		err := r.updateStatus(o, err)
```
Comment (delgod) [inline pkg/controller/pxc/controller.go]: is this function works in reality?

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:-

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-182-170389e

**Pull request merged**